Actions/ReadSettings/action.yaml

```diff
@@ -21,10 +21,6 @@ inputs:
     description: Project folder
     required: false
     default: '.'
-  getProjects:
```

Comment:
This is not so much a comment as it is a question; Do we consider this a breaking change in AL-GO? Generally, is it a breaking change to remove a parameter from an AL-GO-Action?